client-app/core/composables/useLanguages.ts

```diff
@@ -10,134 +10,143 @@ import type { I18n } from "@/i18n";
+    const locale = currentLanguage.value?.twoLetterLanguageName || FALLBACK_LOCALE;
 
-async function loadModuleLocale(i18n: I18n, moduleName: string): Promise<void> {
-  const locale = currentLanguage.value?.twoLetterLanguageName || FALLBACK_LOCALE;
+    try {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+      const [moduleFallbackMessages, moduleMessages] = await Promise.all([
+        locale !== FALLBACK_LOCALE
+          ? import(`../../modules/${moduleName}/locales/${FALLBACK_LOCALE}.json`).catch((error) => {
+              Logger.error(`Fallback locale: ${FALLBACK_LOCALE} for the module ${moduleName} not found`, error);
 
-  try {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    const [moduleFallbackMessages, moduleMessages] = await Promise.all([
-      locale !== FALLBACK_LOCALE
-        ? import(`../../modules/${moduleName}/locales/${FALLBACK_LOCALE}.json`).catch((error) => {
-            Logger.error(`Fallback locale: ${FALLBACK_LOCALE} for the module ${moduleName} not found`, error);
+              return {};
+            })
+          : Promise.resolve({}),
+        import(`../../modules/${moduleName}/locales/${locale}.json`).catch((error) => {
+          Logger.error(`Locale: ${locale} for the module ${moduleName} not found`, error);
 
-            return {};
-          })
-        : Promise.resolve({}),
-      import(`../../modules/${moduleName}/locales/${locale}.json`).catch((error) => {
-        Logger.error(`Locale: ${locale} for the module ${moduleName} not found`, error);
+          return {};
+        }),
+      ]);
 
-        return {};
-      }),
-    ]);
+      const existingMessages = i18n.global.getLocaleMessage(locale as string);
 
-    const existingMessages = i18n.global.getLocaleMessage(locale as string);
+      i18n.global.setLocaleMessage(locale, merge({}, existingMessages, moduleMessages));
 
-    i18n.global.setLocaleMessage(locale, merge({}, existingMessages, moduleMessages));
+      if (locale !== FALLBACK_LOCALE) {
+        const existingFallbackMessages = i18n.global.getLocaleMessage(FALLBACK_LOCALE);
+        i18n.global.setLocaleMessage(FALLBACK_LOCALE, merge({}, existingFallbackMessages, moduleFallbackMessages));
+      }
 
-    if (locale !== FALLBACK_LOCALE) {
-      const existingFallbackMessages = i18n.global.getLocaleMessage(FALLBACK_LOCALE);
-      i18n.global.setLocaleMessage(FALLBACK_LOCALE, merge({}, existingFallbackMessages, moduleFallbackMessages));
+      Logger.debug(`The "${moduleName}" module locale: "${locale}" was loaded successfully.`);
+    } catch (error) {
+      Logger.error(`Error loading the ${moduleName} module locale: "${locale}"`);
     }
-
-    Logger.debug(`The "${moduleName}" module locale: "${locale}" was loaded successfully.`);
-  } catch (error) {
-    Logger.error(`Error loading the ${moduleName} module locale: "${locale}"`);
   }
-}
 
-export function useLanguages() {
+  const fallback = {
```

Comment:
refactoring in the separate pull request

Reply:
Fallback is used at
https://github.com/VirtoCommerce/vc-frontend/blob/f7fce67749006daffb78352e7545f77ec7dc21d5/.storybook/preview.ts#L17
so alternative is to copy-paste it in this PR, which I would like to avoid